include/rocksdb/utilities/options_type.h

```diff
@@ -35,6 +35,7 @@ enum class OptionType {
   kCompactionPri,
   kCompressionType,
   kCompactionStopStyle,
+  kFilePreload,
```

Comment:
Is this type needed or can this just be an Enum?  I rather not see new types added if not required.

Reply:
@mrambacher I will attempt the kEnum.  But I want to point out that there is no current option using that type.  Leaves me unsettled.  I basically copied the code for the kCompaction* as a model previously.

Reply:
I believe I have found the correct example code and implemented equivalently.  All accessory code to my previous method is eliminated.

Reply:
done